Library/Homebrew/help.rb

```diff
@@ -17,21 +17,21 @@ module Help
     HOMEBREW_HELP = <<~EOS
       Example usage:
         brew search [TEXT|/REGEX/]
-        brew info [FORMULA...]
-        brew install FORMULA...
+        brew info [FORMULA|CASK...]
+        brew install FORMULA|CASK...
         brew update
-        brew upgrade [FORMULA...]
-        brew uninstall FORMULA...
-        brew list [FORMULA...]
+        brew upgrade [FORMULA|CASK...]
+        brew uninstall FORMULA|CASK...
+        brew list [FORMULA|CASK...]
 
       Troubleshooting:
         brew config
         brew doctor
-        brew install --verbose --debug FORMULA
+        brew install --verbose --debug FORMULA|CASK
 
       Contributing:
         brew create [URL [--no-fetch]]
-        brew edit [FORMULA...]
+        brew edit [FORMULA|CASK...]
```

Comment:
`brew edit` should work with either no arguments or just one formula or cask right?